pkg/executables/govc.go

```diff
@@ -354,7 +354,7 @@ func (g *Govc) deployTemplate(ctx context.Context, library, templateName, deploy
 		templateInLibraryPath = fmt.Sprintf("/%s", templateInLibraryPath)
 	}
 
-	deployOptsPath, err := g.writer.Write(deployOptsFile, deployOpts, filewriter.PersistentFile)
+	deployOptsPath, err := g.writer.Write(deployOptsFile, []byte(fmt.Sprintf(deployOpts, network)), filewriter.PersistentFile)
```

Comment:
I would vote to move `deployOpts` to be a go template or even just a go struct and then marshall to json